### ISSUE TITLE
Unknown Artist ID3

### ISSUE THREAD

**Issue by trmendes** [AdrienPoupa/VinylMusicPlayer#40]
When I check my artisit list I see there are seven albuns saying they have no artist info but when I open each music using an ID3 Tag Editor or even using the Vinyl's tag editor I see that every tag is filled correctly. 

Using the tag editor or any other mp3 player on my computer I see the artist name and there is no tags missing.

Am I doing something wrong here?

**Comment by trmendes** [comment]
A screenshot of what is happening here.

So this first one shows an album without an Artist name on it.

![screenshot_vinyl_music_player_20180514-062712](https://user-images.githubusercontent.com/16507629/39990075-18b21600-5742-11e8-9e54-0914de8c5d50.png)

So to fix it I open the tag editor to insert a name for this artist and the name is already there.
Anyway I try to change the name and save but nothing changes, The Vinyl Music Player still can't find this artist name,

![screenshot_vinyl_music_player_20180514-062722](https://user-images.githubusercontent.com/16507629/39990079-18da0c78-5742-11e8-8774-c5a2b0fb9a08.png)

I have tried to clean all the app cache and data and what I've got after it is: more artists that before had their artists at the artist list classifed as "unknow artist".

The problem happens no matter if I'm using a mp3 file or an ogg file (I haven't tired with other formatas)
